src/geo/ui/search.js

```diff
@@ -1,31 +1,64 @@
 
+/**
+ *  UI component to place the map in the
+ *  location found by the geocoder.
+ *
+ */
+
 cdb.geo.ui.Search = cdb.core.View.extend({
 
   className: 'cartodb-searchbox',
 
+  _ZOOM_BY_CATEGORY: {
+    'building': 18,
+    'postal-area': 15,
+    'default': 12
+  },
+
   events: {
-    "click input[type='text']":   '_focus',
-    "submit form":                '_submit',
-    "click":                      '_stopPropagation',
-    "dblclick":                   '_stopPropagation',
-    "mousedown":                  '_stopPropagation'
+    "click input[type='text']": '_onFocus',
+    "submit form": '_onSubmit',
+    "click": '_stopPropagation',
+    "dblclick": '_stopPropagation',
+    "mousedown": '_stopPropagation'
   },
 
-  initialize: function() {},
+  options: {
+    searchPin: false,
```

Comment:
Not sure about this neither... making it configurable in the Editor requires viz.json changes, right? I don't think that enabling it by default could cause any actual problem for a user workflow. Probably the users that want this feature are the ones who are not able to use CartoDB.js to enable/disable it, so...